src/operator/tensor/elemwise_binary_broadcast_op.h

```diff
@@ -144,7 +144,7 @@ void BinaryBroadcastCompute(const nnvm::NodeAttrs& attrs,
   int ndim = BinaryBroadcastShapeCompact(inputs[0].shape_, inputs[1].shape_, outputs[0].shape_,
                                          &new_lshape, &new_rshape, &new_oshape);
   if (!ndim) {
-    BinaryCompute<xpu, OP>(attrs, ctx, inputs, req, outputs);
+    ElemwiseBinaryOp::Launch<xpu, OP>(attrs, ctx, inputs, req, outputs);
```

Comment:
It is intentionally called Launch() because it has the ElemwiseBinaryOp:: qualifier.  This is basic overloading semantics.  I don't think there is any confusion here.  In fact, it is launching and not "computing", which implies the old mshadow F<OP>, which according to Eric is being deprecated.  In many places I have removed Compute and replaced with Launch.